cm-mlops/script/convert-ml-model-huggingface-to-onnx/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python -m transformers.onnx --model=${CM_MODEL_HUGG_PATH} ${PWD}
```

Comment:
Can we have an input_mapping dictionary in _cm.json mapping "input":"CM_MODEL_HUGG_PATH" and in customize.py if env.get("CM_MODEL_HUGG_PATH","") == "", we can throw an error. Error can be thrown like "return {'return': 1, 'error': 'CM_MODEL_HUGG_PATH is not set'}"

Reply:
modified 👍🏼

Reply:
Thank you @anandhu-eng. Are we storing the final onnx filepath in an exported ENV variable?

Reply:
On latest commit, its being written into temp-env file as well as stored as ENV variable